api/src/chat/services/conversation.service.ts

```diff
@@ -68,6 +68,9 @@ export class ConversationService extends BaseService<
   ) {
     const msgType = event.getMessageType();
     const profile = event.getSender();
+
+    if (!convo.context) throw new Error('Missing conversation context');
```

Comment:
See previous comment.

Reply:
Yes the capture_vars is defaults empty array
But the schema type of the field don't reflect the value (capture_vars can have an undefined type)
that's why typescript is alerting the possibility to have an undefined type
![image](https://github.com/user-attachments/assets/54aefdba-772a-452a-9fea-e60d13f85d77)